GW2EIEvtcParser/EIData/Actors/Minions.cs

```diff
@@ -271,6 +271,21 @@ public override IEnumerable<CastEvent> GetIntersectingCastEvents(ParsedEvtcLog l
         }
         return CastEvents.Where(x => KeepIntersectingCastLog(x, start, end));
     }
+
+    public override IEnumerable<AnimatedCastEvent> GetAnimatedCastEvents(ParsedEvtcLog log, long start, long end)
```

Comment:
That implement does not make sense, minions are a group of multiple AgentItems, those will return events for only a single agent within the pack.
Also that would require events to be sorted by time at each call, which is going to be very inneficient. I think you can limit those APIs to SingleActor.